wpilibj/src/main/java/edu/wpi/first/wpilibj/experimental/command/WaitUntilCommand.java

```diff
@@ -0,0 +1,43 @@
+package edu.wpi.first.wpilibj.experimental.command;
+
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ * A command that does nothing but ends after a specified match time or condition.  Useful for
+ * CommandGroups.
+ */
+public class WaitUntilCommand extends SendableCommandBase {
+
+  private final BooleanSupplier m_condition;
+
+  /**
+   * Creates a new WaitUntilCommand that ends after a given condition becomes true.
+   *
+   * @param condition the condition to determine when to end
+   */
+  public WaitUntilCommand(BooleanSupplier condition) {
+    m_condition = condition;
+  }
+
+  /**
+   * Creates a new WaitUntilCommand that ends after a given match time.
+   *
+   * @param time the match time after which to end, in seconds
+   */
+  public WaitUntilCommand(double time) {
```

Comment:
Are we ok encouraging this behavior when we do not guarantee the timing from the DS?

Reply:
I've added a caveat in the javadoc about this.